deploy/crd.yaml

```diff
@@ -91,6 +91,9 @@ spec:
                   type: string
             savepointPath:
               type: string
+            maxCheckpointRestoreAgeSec:
```

Comment:
According to the [kubernetes api guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units), the unit in the duration field name should not be abbreviated (so `Seconds` instead of `Sec`).